Atlas/backend/atlas/views.py

```diff
@@ -42,15 +41,12 @@ class image_media_item(ImageInterceptorMixin,generics.CreateAPIView):
     queryset = image_item.objects.all()
     serializer_class = image_media_item_serializer
     def create(self, request, *args, **kwargs):
-        try:
-            for image_item_data in request.data['images']:
-                 request.data['cultural_heritage_item'] = self.current_heritage_item.pk
-                 for k,v in image_item_data.items():
-                     request.data[k] = v
-                 result =super(image_media_item, self).create(request, *args, **kwargs)
-            return result
-        except:
-           return Response("images must be a list.", status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Why did you remove this line ? If there is a problem while creating images  it will give internal server error if you remove this line.   However with this line clients will get HTTP 400 error.